### ISSUE TITLE
packages check - not parsing parameters 

### ISSUE THREAD

**Issue by noahjl** [CiscoCloud/distributive#27]
Validating some of the checks and noticing a consistent fail for repoExists checks - 

```json
{
    "Name": "chronos health check",
    "Notes": "For use with Microservice Infrastructure",
    "Checklist" : [
        {
            "Name": "mesosphere repo present in yum",
            "Check": "repoExistsURI",
            "Parameters": ["yum","http://repos.mesosphere.io/el/7/noarch/"]
        },
....
```

based on packages.go it seems that the baseurl or name in /etc/yum.repos.d/mesosphere.repo should fit the parameters. But they return errors:

```bash
Repo with given Url not found:
        Specified: http://repos.mesosphere.io/el/7/$basearch/
        Actual: []
```

Using repoExists and the "name=" in mesosphere.repo 

```bash
Repo with given Name not found:
        Specified: Mesosphere Packages for EL 7 - noarch
        Actual: []
``` 

Using the microservices-infrastructure vagrant box for testing.

I've also installed the RPM of distributive 0.1.

**Comment by siddharthist** [comment]
Awesome, thanks for testing! This one is particularly difficult, as I don't really have three package managers on my system. In fact, the code is only looking through /etc/yum.conf. Can you send me both your /etc/yum.conf and /etc/yum.repos.d?

**Comment by siddharthist** [comment]
TODO: Improve repo check to read from information outputted by yum & deb.

**Comment by siddharthist** [comment]
Fixed by ca099df4a606531607b7d25bc5e8d8b70bc3e0ba
